prov/psm/src/psmx_domain.c

```diff
@@ -307,12 +304,7 @@ int psmx_domain_open(struct fid_fabric *fabric, struct fi_info *info,
 	if (psmx_domain_enable_ep(domain_priv, NULL) < 0)
 		goto err_out_finalize_mq;
 
-	err = fastlock_init(&domain_priv->poll_lock);
-	if (err) {
-		FI_WARN(&psmx_prov, FI_LOG_CORE,
-			"pthread_spin_init returns %d\n", err);
-		goto err_out_finalize_mq;
-	}
+	fastlock_init(&domain_priv->poll_lock);
```

Comment:
I will rework this one now we have the update from PR#1430. There are a few other places that the return value can be checked.